viewer/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "viewer",
-  "version": "1.44.1",
+  "version": "2.4.1",
```

Comment:
I think it's fine to include this version bump in the lock file, but I would normally think this would happen in response to an updated library or some other change in the `package.json` file. Is there a corollary change to that file you want to include, or maybe not include this? 

And also - leaving it in is fine too, since it's a legit reflection of the current version of the viewer.

Reply:
she could have ran `npm install` which will auto update the version, i think its fine to include since it reflects the current version :shrug:

Reply:
yep, the lock file version is being brought back into sync with `package.json`. i guess as a best practice we should be running `npm install` after bumping our own app version...